prime-router/metadata/organizations-local.yml

```diff
@@ -28,7 +47,7 @@
             type: SFTP
             host: sftp
             port: 22
-            filePath: ./upload
+            filePath: ./upload_prod
```

Comment:
This wont work without a change to the SFTP server - right now we're writing to ./upload will need to setup foo user to write to /upload/prod or /upload/test